makefiles/tools/bossa.inc.mk

```diff
@@ -1,7 +1,14 @@
 BOSSA_VERSION ?= 1.9
 FLASHFILE ?= $(BINFILE)
 FLASHER ?= $(RIOTTOOLS)/bossa-$(BOSSA_VERSION)/bossac
-FFLAGS  ?= -p $(PROG_DEV) -o $(ROM_OFFSET) -e -i -w -v -b -R $(FLASHFILE)
+FFLAGS_OPTS ?=
+
+# Only use ROM_OFFSET with Bossa version 1.9
```

Comment:
Why?

Reply:
For some reason the Arduino (quick-n-dirty) fork of Bossa already takes in to account the ROM_OFFSET when flashing the firmware for nrf52. So we have to only take it into account with 1.9 which is used for samdx CPUs.

Reply:
Ok, I ended up tracking the original PR where you did this. I think the in code comment could be more clear. But OK